SweetsFactory/src/app/app.module.ts

```diff
@@ -1,23 +1,35 @@
 import { NgModule } from '@angular/core';
-import { BrowserModule }  from '@angular/platform-browser';
+import { BrowserModule } from '@angular/platform-browser';
 import { AppComponent } from './start/app.component';
+import { FormsModule } from '@angular/forms';
 
-import { NavComponent } from './shared/navbar.component';
+import { NavComponent } from './shared/navbar/navbar.component';
+import { SearchBarComponent } from './shared/searchbar/search.component';
+import { AccountComponent } from './shared/account/account.component';
+import { MenuItemComponent } from './shared/menuItem/menuItem.component';

```

Comment:
Why do you implement here a menu item instead of the whole menu? (as component)

Reply:
So that i could create the menu more dynamic